packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.controller.js

```diff
@@ -71,7 +78,38 @@ export default class TelecomPackMigrationOffersCtrl {
   }
 
   selectOffer(offer) {
```

Comment:
The method seems to delete services, so the name should be changed

Reply:
No, this method is called when you click on the "Choose this offer" button. We then need to update the list of services to be deleted from the additional options selected in this same page. So the name is correct.